docs/user-guide/map-catalog.md

```diff
@@ -0,0 +1,37 @@
+# Map Catalog
+*******************
+
+The **Map Catalog** is an extension added to the context to browse the catalog of the user map. The <img src="../img/button/user-extensions-button.jpg" class="ms-docbutton"/> button, present in *Burger Menu* <img src="../img/button/burger.jpg" class="ms-docbutton" />, provide to the user the list of the maps available on the [Homepage](https://mapstore.geo-solutions.it/mapstore/#/). 
+
+<img src="../img/map-catalog/map-catalog-panel.jpg" class="ms-docimage"/>
+
+The *Map Catalog* panel opens allows the user to select a map and loaded it in the same browser page, as follows:
+
+<img src="../img/map-catalog/context-to-map.gif" class="ms-docimage"/>
+
+!!! note
+    The added map replaces the context, in this way the context extensions are no more available.
+
+For each map in the *Map Catalog* list the following buttons are displayed:
+
+<img src="../img/map-catalog/buttons-on-map-catalog.jpg" class="ms-docimage"/>
+
+* The **Delete** button <img src="../img/button/delete_button.jpg" class="ms-docbutton" /> allows the user to remove the map
+
+!!! warning
+    The map deleted from the *Map Catalog* remains available on the MapStore [Homepage](https://mapstore.geo-solutions.it/mapstore/#/).
```

Comment:
**Remove this warning**, the map list is the same betweek the homepage and the MapCatalog

>If you found something like this, generally speaking this is a bug and not something to document! It is working well anyway, just tested, so remove the warning.